src/devices/acurite.c

```diff
@@ -506,7 +575,7 @@ static int acurite_txr_decode(r_device *decoder, bitbuffer_t *bitbuffer)
                     "channel",              NULL,           DATA_STRING, &channel_str,
                     _X("battery_ok","battery_low"), "",     DATA_INT,    _X(!battery_low,battery_low),
                     "temperature_C",        "Temperature",  DATA_FORMAT, "%.1f C", DATA_DOUBLE, tempc,
-                    "humidity",             "Humidity",     DATA_INT,    humidity,
+                    "humidity",             "Humidity",     DATA_FORMAT, "%u %%", DATA_INT,    humidity,
```

Comment:
See comment above, would prefer not to have the Atlas PR change the output format for other devices